docs/en-US/component/link.md

```diff
@@ -53,19 +53,21 @@ link/with-icon
 
 :::
 
-## Attributes
-
-| Name      | Description                         | Type                  | Accepted Values                                       | Default |
-| --------- | ----------------------------------- | --------------------- | ----------------------------------------------------- | ------- |
-| type      | type                                | string                | primary / success / warning / danger / info / default | default |
-| underline | whether the component has underline | boolean               | —                                                     | true    |
-| disabled  | whether the component is disabled   | boolean               | —                                                     | false   |
-| href      | same as native hyperlink's `href`   | string                | —                                                     | -       |
-| icon      | icon component                      | `string \| Component` | —                                                     | -       |
-
-## Slots
-
-| Name | Description               |
-| ---- | ------------------------- |
-| —    | customize default content |
-| icon | customize icon component  |
+## API
+
+### Attributes
+
+| Name      | Description                         | Type Values                                                                       | Default |
```

Comment:
## Question

Is there a reason they are `Type Values` ​​and not `Types`?

Reply:
sry, this was my mistake.